atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -206,8 +207,16 @@ def push_tarball_to_pulp(self, image_names, repo_prefix="redhat-"):
                 p.upload(outfile.name)
         except:
             self.log.debug("Error on creating deduplicated layers tar", exc_info=True)
-            self.log.info("Falling back to full tar upload")
-            p.upload(self.filename)
+            try:
+                if file_extension == '.tar':
+                    raise RuntimeError("tar is already compressed")
```

Comment:
I don't think tar by itself is compressed. Should this check for `.gz` file extension?